packages/student_ids/earthmover.yaml

```diff
@@ -45,6 +45,8 @@ config:
           -- and api_year=${SNOWFLAKE_API_YEAR}
           -- and assessment='${ASSESSMENT_BUNDLE}'
       )
+
+    NO_MATCH_VALUE: __no_match__
```

Comment:
Thoughts on removing this from parameters and just hardcoding the value below? I know it was originally conceived this way but I don't really see why this is something the caller should be setting